test/README.md

```diff
@@ -82,6 +82,45 @@ environment.
    docker is used only for running drenv tests locally. You can use
    podman for building and running containers locally.
 
+1. Install `podman`
+
+   ```
+   sudo dnf install podman
+   ```
+
+   ramenctl depends on podman, hence it needs to be installed.
+
+   See [Installing podman](https://podman.io/docs/installation) for other options.
+
+## Troubleshooting drenv issues
+
+### podman commands are failing for rootless access
+
+```
+$ podman version
+ERRO[0000] running /usr/bin/newuidmap 61879 0 1000 1 1 100000 65536: newuidmap:
+open of uid_map failed: Permission denied
+Error: cannot set up namespace using "/usr/bin/newuidmap": exit status 1
+```
+
+While running with root access it works fine.
+
+```
+$ sudo podman version
+Client:       Podman Engine
+Version:      4.6.2
+API Version:  4.6.2
+Go Version:   go1.19.12
+Built:        Tue Aug 29 01:08:31 2023
+OS/Arch:      linux/amd64
```

Comment:
But this does not add much value, we can replace this with a short note that has more info:

Reported with podman 4.6.2 on Fedora 37.

Reply:
Made it a short and added the note.